apps/swagger/package.json

```diff
@@ -14,7 +14,7 @@
   "dependencies": {
     "highlight.js": "^11.6.0",
     "isarray": "2.0.5",
-    "next": "^13.2.1",
+    "next": "~13.2.1",
```

Comment:
Lock swagger to 13.2, no reason for 13.4

Reply:
Why not upgrade as well? That way we prevent two next.js versions on the lock file.

Reply:
I want to isolate this for now so I can more thoroughly test separate next versions - so we can stagger things in the future when we grow and have more apps.